tests/js/server/aql/aql-graph-traverser-optimizeIn.js

```diff
@@ -0,0 +1,260 @@
+/* jshint esnext: true */
+/* global AQL_EXECUTE, AQL_EXPLAIN, AQL_EXECUTEJSON */
+
+// //////////////////////////////////////////////////////////////////////////////
+// / @brief Spec for the AQL FOR x IN GRAPH name statement
+// /
+// / @file
+// /
+// / DISCLAIMER
+// /
+// / Copyright 2014 ArangoDB GmbH, Cologne, Germany
+// /
+// / Licensed under the Apache License, Version 2.0 (the "License");
+// / you may not use this file except in compliance with the License.
+// / You may obtain a copy of the License at
+// /
+// /     http://www.apache.org/licenses/LICENSE-2.0
+// /
+// / Unless required by applicable law or agreed to in writing, software
+// / distributed under the License is distributed on an "AS IS" BASIS,
+// / WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// / See the License for the specific language governing permissions and
+// / limitations under the License.
+// /
+// / Copyright holder is ArangoDB GmbH, Cologne, Germany
+// /
+// / @author Michael Hackstein
+// / @author Copyright 2015, ArangoDB GmbH, Cologne, Germany
+// //////////////////////////////////////////////////////////////////////////////
+
+'use strict';
+
+const jsunity = require('jsunity');
+const {assertEqual, assertTrue, assertFalse, fail} = jsunity.jsUnity.assertions;
+
+var _ = require('lodash');
+const internal = require('internal');
+const db = internal.db;
+const errors = require('@arangodb').errors;
+const gm = require('@arangodb/general-graph');
+const vn = 'UnitTestVertexCollection';
+const en = 'UnitTestEdgeCollection';
+const isCluster = require('@arangodb/cluster').isCluster();
+const removeCost = require('@arangodb/aql-helper').removeCost;
```

Comment:
Only this testfile needs the removeCost method. All other files are not using it, but they do require it. 
Please check if that might be true for other requires here as well.